sandbox/src/routes/retrieveClinicalInformation.js

```diff
@@ -0,0 +1,39 @@
+const mockResponseProvider = require('../services/mockResponseProvider')
+const businessFunctionValidator = require('../services/businessFunctionValidator')
+
+module.exports = [
+  /**
+   * Sandbox implementation for retrieveClinicalInformation endpoint
+   */
+  {
+    method: 'POST',
+    path: '/STU3/v1/ReferralRequest/{ubrn}/$ers.generateCRI',
+    handler: (request, h) => {
+
+      const allowedBusinessFunctions = ["REFERRING_CLINICIAN", "REFERRING_CLINICIAN_ADMIN", "SERVICE_PROVIDER_CLINICIAN"]
+
+      const validationResult = businessFunctionValidator.validateBusinessFunction(request, h, allowedBusinessFunctions)
+      if (validationResult) {
+        return validationResult
+      }
+
+      // Simply checking if ubrn is supplied
+      if (request.params.ubrn) {
+
+        const { responsePath, responseCode, filename } = mockResponseProvider.getExampleResponseForRetrieveClinicalInformation();
+        if (responsePath !== undefined && responseCode !== undefined) {
+          return h.file(responsePath, {
+            mode: 'attachment',
+            filename: filename
+          }).code(responseCode);
+        }
+      }
+
+      return h.file('SandboxErrorOutcome.json').code(422);
+
+
+    }, options: {
```

Comment:
Why are these additional options required? Are these being included in the other sandbox endpoints?

Reply:
I think its a left over config that I tried to make the file download work, i suspect its not needed I'll investigate